### ISSUE TITLE
Sometimes sends null bytes in message content

### ISSUE THREAD

**Issue by timabbott** [zulip/zulip-mobile#1248]
We've been getting some exceptions in production in the send_message code path of this form:

`ValueError: A string literal cannot contain NUL (0x00) characters.`

I'm going to fix this in the server to be a 400 type error, since it's an invalid request from the client, not a 500, but these are definitely being sent by the mobile app.  

I'm not sure how the app generates a message body that contains null characters; if adding logging on the server side would be useful, that's doable (it's possible these are effectively empty messages).

**Comment by borisyankov** [comment]
I will try to figure out it out.
Having the user-agent string would be useful though to detect platform and version.
We haven't fixed any issue like this, so likely this exists in the latest version too.

**Comment by timabbott** [comment]
Yeah, the part of the error email reporting that would have the app version are currently broken, so I can't get that, but we'll get it if it keeps happening after we fix that.
